apps/web/src/components/AcronymTooltip.tsx

```
@@ -0,0 +1,174 @@
+ */
+export function AcronymTooltip({ text, definition, className = '' }: AcronymTooltipProps) {
+  const [isVisible, setIsVisible] = useState(false);
+  const [position, setPosition] = useState<'top' | 'bottom'>('top');
+  const tooltipRef = useRef<HTMLSpanElement>(null);
+  const triggerRef = useRef<HTMLSpanElement>(null);
+
+  const fullText = definition || ACRONYM_DICTIONARY[text] || ACRONYM_DICTIONARY[text.toUpperCase()];
+  
+  if (!fullText) return <span>{text}</span>;
+
+  useEffect(() => {
```

Biome (2.4.4)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)